src/api/spec/controllers/kerberos_login_spec.rb

```diff
@@ -0,0 +1,98 @@
+require 'rails_helper'
+require 'gssapi'
+
+RSpec.describe 'Kerberos login', vcr: false, type: :request do
```

Comment:
Please create a new folder to follow rspec conventions: `src/api/spec/requests/kerberos_login_spec.rb`